livre/eq/render/shaders/fragAxis.glsl

```diff
@@ -0,0 +1,55 @@
+#version 420
+
+layout( rgba32f ) uniform image2DRect renderTexture;
+
+uniform ivec4 viewport;
+uniform mat4 invProjectionMatrix;
+uniform vec2 depthRange;
+
+flat in vec3 normal;
+flat in vec3 normal2;
+in vec4 color;
+flat in uint type;
+
+// http://www.opengl.org/wiki/Compute_eye_space_from_window_space.
+vec4 calcPositionInEyeSpaceFromWindowSpace( vec4 windowSpace )
```

Comment:
I really don't get it... I need to multiply vertex coordinate by modelViewProjectionMatrix otherwise my fragments positions will be wrong.